nni/algorithms/hpo/dngo_tuner/config.yml

```diff
@@ -0,0 +1,18 @@
+authorName: Ayan Mao
+experimentName: arch2vec_dngo_general_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 100
+trainingServicePlatform: local  # choices: local, remote, pai
+searchSpacePath: search_space.json
+useAnnotation: false
+tuner:
+  codeDir: nni/examples/tuners/dngo_tuner
+  classFileName: dngo_tuner.py
+  className: DngoTuner
+  # Any parameter need to pass to your tuner class __init__ constructor
+  # can be specified in this optional classArgs field, for example
+trial:
```

Comment:
I think this config should be put into example folder, and should be written in V2 format.

Reply:
agree